.travis.yml

```diff
@@ -3,17 +3,25 @@ sudo: false
 cache: bundler
 script:
 - bundle exec rake spec
-rvm:
-- 2.0.0
-- 2.1.5
-- 2.4.2
 matrix:
   fast_finish: true
+  include:
+  - rvm: 2.0.0
+    bundler_args: --without release
+  - rvm: 2.1.5
+    bundler_args: --without release
```

Comment:
2.1.5 will be used to release now.

Reply:
but we only need the group to generate the changelog, we don't require it witin travis

Reply:
Ah right. So why do you use `include` and not just `rvm:` list? IIRC you can set `bundler_args` at the top level as well.

Reply:
I tried it at multiple levels but could not find the correct one.

Reply:
https://github.com/theforeman/puppet-foreman/blob/d33e10c2d157c392b1289215cb192b00d2e30c98/.travis.yml#L81

Reply:
updated